apps/web/src/hooks/useRoutes.ts

```diff
@@ -9,33 +9,31 @@ const getRoutes = (lang: Locales) => {
   const adminUrl = path.join(baseUrl, "admin");
   const userUrl = (userId: string) => path.join(baseUrl, "users", userId);
   const applicationParam = (applicationId?: string) =>
-    applicationId ? path.join(`?applicationId=${applicationId}`) : "";
+    applicationId ? `?applicationId=${applicationId}` : "";
```

Comment:
Could you check the Edit Skills and Experience link?  Looks like we got an extra slash.
![image](https://user-images.githubusercontent.com/8978655/226611107-aab15b42-5b5f-4255-b8ab-51a7da40aa87.png)

Reply:
Good catch! Fixed in [remove trailing slash from edit skills link](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6009/commits/b224e58c4b100b86c731bcdc2cc4993313188be1)